FirebaseAdmin/FirebaseAdmin/Auth/UploadAccountResponse.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace FirebaseAdmin.Auth
+{
+    /// <summary>
+    /// Represents the response from identity toolkit for a user import request.
+    /// </summary>
+    public class UploadAccountResponse
```

Comment:
internal sealed

Actually, consider just nesting this in `FirebaseUserManager`. Doesn't look like we need a top-level class for such a small thing.